terraform-modules/aws/vpc/variables.tf

```
@@ -74,3 +74,35 @@ variable "external_nat_ip_ids" {
   type        = list(string)
   default     = []
 }
+
+#Default Security Group Management (Default: secure)
+variable "manage_default_security_group" {
+  description = "Should be true to adopt and manage default security group"
+  type        = bool
+  default     = true
+}
+
+variable "default_security_group_name" {
+  description = "Name to be used on the default security group	"
+  type        = string
+  default     = "default"
+}
+
+variable "default_security_group_egress" {
+  description = "List of maps of egress rules to set on the default security group"
+  type        = list(map(string))
+  default     = []
```

```suggestion
  default     = ["0.0.0.0/0"]
```

The sensible default would be to allow everything out.